static/sass/_pattern_docs.scss

```diff
@@ -1,5 +1,5 @@
 @mixin anbox-p-docs {
-  .l-docs {
+  .l-doc {
```

Comment:
Had to change this, due to conflict with vanilla class `l-docs`: https://vanillaframework.io/docs/layouts/documentation